spec/01-unit/010-router_spec.lua

```diff
@@ -1681,64 +1681,89 @@ describe("Router", function()
 
     describe("slash handling", function()
       local checks = {
-        -- upstream url    paths            request path    expected path           strip uri
+        -- upstream url    paths           request path    expected path           strip uri
```

Comment:
Unrelated to this PR, but I truly wish the order of those were: request path, route path, upstream url, expected url... Even despite the fact that the upstream URL is built from upstream+path+request. If anybody with some time on their hands is reading this...

Reply:
yes, I had that itch as well 😄